fuzzing/fuzz/.gitignore

```diff
@@ -0,0 +1,2 @@
+/artifacts/
```

Comment:
I didn't know we could have `.gitignore` files in sub folders. TIL.

Reply:
Yeah, it can also be useful if you need a folder to exist: <https://github.com/rinja-rs/rinja/tree/master/testing/folder-config.toml>. Normally does not allow empty folder, so you can add that one hidden file.